ballerina/websocket_sync_client.bal

```diff
@@ -191,8 +191,9 @@ public isolated client class Client {
 
     # Reads text messages in a synchronous manner.
     #
+    # + targetType - The payload type (sybtype of `anydata`), which is expected to be returned after data binding
     # + return  - The text data sent by the server or a `websocket:Error` if an error occurs when receiving
-    remote isolated function readTextMessage() returns string|Error = @java:Method {
+    remote isolated function readTextMessage(TargetType targetType = <>) returns targetType|Error = @java:Method {
```

Comment:
Since the `TargetType` contains only the typedesc of `anydata`, shall we use it directly here rather declaring a new type. So docs will be clear and no need to check the definition of `TargetType` by clicking a new link